static/app/views/issueDetails/streamline/useOrganizationFlagLog.tsx

```diff
@@ -0,0 +1,19 @@
+import type {Organization} from 'sentry/types/organization';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import type {RawFlagData} from 'sentry/views/issueDetails/streamline/featureFlagUtils';
+
+export function useOrganizationFlagLog({
+  organization,
+  query,
+}: {
+  organization: Organization;
+  query: Record<string, any>;
+}) {
+  return useApiQuery<RawFlagData>(
+    [`/organizations/${organization.slug}/flags/logs/`, {query}],
+    {
+      staleTime: 0,
+      enabled: organization.features?.includes('feature-flag-ui'),
```

Comment:
should be always available
```suggestion
      enabled: organization.features.includes('feature-flag-ui'),
```